locustfile.py

```diff
@@ -0,0 +1,225 @@
+                             filters={"route": route},
+                             include=["route", "vehicle", "service",
+                                      "predictions"])
+
+    # station schedules/predictions
+
+    @task(len(solari_screen_stations))
+    def get_station_predictions(self):
+        for stop in self.solari_screen_stations:
+            self.api_request("/predictions",
+                             name="/predictions (Solari screens)",
+                             filters={"stop": stop},
+                             include=["route", "stop", "trip", "trip.stops",
+                                      "vehicle", "alerts"],
+                             sort="departure_time")
+
+    @task(2 * len(eink_screen_stations))
+    def get_station_schedules(self):
+        direction = randint(0, 1)
+        self.api_request("/schedules",
+                         name="/schedules (e-ink screens)",
+                         filters={"direction_id": direction,
+                                  "stop": self.eink_screen_stations},
+                         include=["route", "stop", "trip"],
+                         sort="departure_time")
+
+    @task
+    def get_back_bay_schedules(self):
+        self.api_request("/schedules",
+                         name="/schedules (Commuter Rail @ Back Bay)",
+                         filters={"route": ["CR-Worcester", "CR-Franklin",
+                                            "CR-Needham", "CR-Providence"],
+                                  "stop": "place-bbsta"},
+                         include=["route", "stop", "trip"],
+                         sort="departure_time")
+
+    @task
+    def get_commuter_rail_station_predictions(self):
+        self.api_request("/predictions",
+                         name="/predictions (Commuter Rail stations)",
+                         filters={"route_type": 2,
+                                  "stop": ["place-sstat", "place-north",
+                                           "place-bbsta"]},
+                         include=["trip", "schedule", "stop"])
+
+    # route schedules/predictions
+
+    @task(len(green_line_routes))
+    def get_green_line_schedules_with_predictions(self):
+        for route in self.green_line_routes:
+            self.api_request("/schedules",
+                             name="/schedules (Green Line branches)",
+                             filters={"route": route},
+                             include=["prediction", "stop"])
+
+    @task(len(heavy_rail_routes))
+    def get_heavy_rail_schedules_with_predictions(self):
+        for route in self.heavy_rail_routes:
+            self.api_request("/schedules",
+                             name="/schedules (heavy rail)",
+                             filters={"route": route},
+                             include=["stop", "trip", "prediction"],
+                             sort="arrival_time")
+
+    @task(len(heavy_rail_routes))
```

Comment:
This seems like it'll run N*N API calls for the heavy rail routes? Or does `@task(count)` not run the given task `count` times?